lib/kubernetes-deploy/runner.rb

```diff
@@ -187,16 +187,11 @@ def server_major_version
 
     # Inspect the file referenced in the kubectl stderr
     # to make it easier for developer to understand what's going on
-    def find_bad_file_from_kubectl_output(stderr)
+    def find_bad_files_from_kubectl_output(stderr)
       # Output example:
       # Error from server (BadRequest): error when creating "/path/to/configmap-gqq5oh.yml20170411-33615-t0t3m":
-      match = stderr.match(%r{BadRequest.*"(?<path>\/\S+\.ya?ml\S*)"})
-      return unless match
-
-      path = match[:path]
-      if path.present? && File.file?(path)
-        File.read(path)
-      end
+      matches = stderr.scan(%r{"(?<path>\/\S+\.ya?ml\S*)"})
```

Comment:
The example above shows how the error looks like for a single failure. Can you update the example for multiple failures?

I'm asking because I find it extremely helpful to be able to see an example of the string that the regexp expects.